meerk40t/main.py

```diff
@@ -24,7 +25,14 @@
 APPLICATION_VERSION = "0.7.5-beta1"
 
 if not getattr(sys, "frozen", False):
-    APPLICATION_VERSION += " src"
+    # If .git directory does not exist we are running from a package like pypi
+    # Otherwise we are running from source
+    if os.path.isdir(sys.path[0] + "/.git"):
+        APPLICATION_VERSION += " git"
+    elif os.path.isdir(sys.path[0] + "/.github"):
+        APPLICATION_VERSION += " src"
+    else:
+        APPLICATION_VERSION += " pkg"
```

Comment:
It doesn't seem like pkg is useful for mato folks or .git. And I'm not sure those folders existing mean this stuff.

Reply:
These suffixes were introduced so that the bug reports had information about the type of execution. 

I have checked the Pypi distributions and the Github downloads and this does seem to be a valid way to distinguish between Git, Pypi and Github download executions.